tlscanary/template/js/report_page.js

```diff
@@ -85,98 +83,89 @@ function resizeChartCanvas()
 function getPieChartData (uriList, fieldName) {
   var chartFields = [];
   var strTable = "";
-  for (var i=0;i<uriList.length;i++)
+  for (var i = 0;i < uriList.length;i++)
   {
     var labelString = uriList[i][fieldName].toString();
     if (strTable.indexOf(labelString) == -1 )
     {
       strTable += labelString;
       chartFields.push (
         {
-          label:labelString,
-          value:1
-        });
+          label: labelString,
+          value: 1
+        }
+      );
     } else {
-      for (var j=0;j<chartFields.length;j++)
+      for (var j = 0;j < chartFields.length;j++)
       {
-        if (chartFields[j].label == labelString)
+        if (chartFields[j].label === labelString)
         {
           chartFields[j].value++;
         }
       }
     }
   }
   var colorArray = returnColorArray(chartFields.length);
-  for (var i=0; i<chartFields.length; i++) {
+  for (var i = 0;i < chartFields.length; i++) {
     chartFields[i].color = colorArray[i];
   }
   return chartFields;
-};
+}
 
 // Credit here goes to http://krazydad.com/tutorials/makecolors.php
 function byte2Hex(n) {
-  var nybHexString = "0123456789ABCDEF";
+  const nybHexString = "0123456789ABCDEF";
   return String(nybHexString.substr((n >> 4) & 0x0F,1)) + nybHexString.substr(n & 0x0F,1);
 }
 
 function RGB2Color(r,g,b) {
   return '#' + byte2Hex(r) + byte2Hex(g) + byte2Hex(b);
 }
 
-function returnColorArray (n) {
-  var a = [];
-  var frequency = 0.3;
+function returnColorArray(n) {
+  var colorArray = [];
```

Comment:
I think this can be const, too.

Reply:
Mea culpa - you are right. Seems to be fine. I spoke too soon. :)